benchmark/triangles.ipynb

```diff
@@ -0,0 +1,367 @@
+{
```

Comment:
I'm not sure I like the idea of notebooks in the `benchmark/` directory. Perhaps we can put them in docs and create a section for "miscellaneous notes"?